data_steward/cdr_cleaner/cleaning_rules/drop_cope_duplicate_responses.py

```diff
@@ -75,7 +75,8 @@
     JOIN
       `{{project}}.{{dataset}}.{{cope_survey_version_table}}` cope
     ON
-      obs.questionnaire_response_id = cope.questionnaire_response_id )
+      obs.questionnaire_response_id = cope.questionnaire_response_id 
+      AND obs.person_id = cope.participant_id)
```

Comment:
This looks good, but I'm curious what it was doing before you added this. Was it still returning the correct rows?

Reply:
@jp3477 ran the integration tests, they passed in CircleCi as well.